released/packages/coq-mathcomp-algebra-tactics/coq-mathcomp-algebra-tactics.1.2.7/opam

```diff
@@ -0,0 +1,45 @@
+opam-version: "2.0"
+maintainer: "kazuhiko.sakaguchi@inria.fr"
+version: "dev"
+
+homepage: "https://github.com/math-comp/algebra-tactics"
+dev-repo: "git+https://github.com/math-comp/algebra-tactics.git"
+bug-reports: "https://github.com/math-comp/algebra-tactics/issues"
+license: "CECILL-B"
+
+synopsis: "Ring, field, lra, nra, and psatz tactics for Mathematical Components"
+description: """
+This library provides `ring`, `field`, `lra`, `nra`, and `psatz` tactics for
+the Mathematical Components library. These tactics use the algebraic
+structures defined in the MathComp library and their canonical instances for
+the instance resolution, and do not require any special instance declaration,
+like the `Add Ring` and `Add Field` commands. Therefore, each of these tactics
+works with any instance of the respective structure, including concrete
+instances declared through Hierarchy Builder, abstract instances, and mixed
+concrete and abstract instances, e.g., `int * R` where `R` is an abstract
+commutative ring. Another key feature of Algebra Tactics is that they
+automatically push down ring morphisms and additive functions to leaves of
+ring/field expressions before applying the proof procedures."""
+
+build: [make "-j%{jobs}%" "build"]
+run-test: [make "-j%{jobs}%" "test-suite"]
+install: [make "install"]
+depends: [
+  ("coq" {>= "8.20" & < "9~"} & "coq-elpi" {>= "2.2.0" & < "3~"}
+  | "coq-core" {>= "9.0" & < "9.2~"} & "coq-stdlib" & "coq-elpi" {>= "2.2.0"})
+  "coq-mathcomp-ssreflect" {>= "2.4" & < "2.5~" | = "dev"}
+  "coq-mathcomp-algebra"
+  "coq-mathcomp-zify" {>= "1.5.0"}
+]
+
+tags: [
+  "logpath:mathcomp.algebra_tactics"
+]
+authors: [
+  "Kazuhiko Sakaguchi"
+  "Pierre Roux"
+]
+url {
+  src: "https://github.com/math-comp/algebra-tactics/archive/refs/tags/1.2.6.tar.gz"
```

Comment:
1.2.6?

Reply:
Arg, these opam files are quite painful to edit, forgot that part indeed.